apps/frontend/src/components/cards/controltable/ControlRowDetails.vue

```diff
@@ -10,11 +10,33 @@
 
           <v-tab-item value="tab-test">
             <div class="pa-4">
-              <div v-if="caveat || justification || rationale || comments">
+              <div
+                v-if="
+                  caveat ||
+                  justification ||
+                  rationale ||
+                  comments ||
+                  errorMessage !== ''
```

Comment:
similar to the other ones blocks, unless we're explicitly only checking for empty string, we should probably do the 'string as boolean' thing so that it handles null/undefined as well

Reply:
Since we aren't checking for a value from the `control.hdf` object and rather a custom string based on a condition, wouldn't we want to still check for an empty string unless we make the fail condition a null/undefined

Reply:
Re-reading the code, I see what you mean w/r to that function basically returning an enum of [warning | empty-string].  However, in that case, it makes even more sense to just do v-if="stuff || comments || errorMessage" cause "errorMessage !== ''" is semantically equivalent to "errorMessage (as a boolean expression)" due to the empty string being falsy.